website/content/api-docs/secret/pki/issuance.mdx

```diff
@@ -367,6 +370,141 @@ $ curl \
 }
 ```
 
+### List ACME Account Keys
+
+This endpoint returns a list of ACME account key identifiers.
+
+| Method | Path                           |
+|:-------|:-------------------------------|
+| `LIST` | `/pki/acme/mgmt/account/keyid` |
+
+#### Sample request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request LIST \
+    http://127.0.0.1:8200/v1/pki/acme/mgmt/account/keyid
+```
+
+#### Sample response
+
+```
+{
+  "data": {
+    "keys": [
+      "2ea9859a-eba8-ff24-cd03-2a51639fc7d5"
+    ]
+  }
+}
+```
+
+### Get ACME Account Info
+
+This endpoint returns the account information for a given ACME account key along with
+the associated orders and certificate details issued within the account.
+
+| Method | Path                                   |
+|:-------|:---------------------------------------|
+| `GET`  | `/pki/acme/mgmt/account/keyid/:key_id` |
+
+#### Parameters
+
+- `key_id` `(string: <required>)` - The id of the ACME account to read.
+  This is part of the request URL.
+
+#### Sample request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/pki/acme/mgmt/account/keyid/2ea9859a-eba8-ff24-cd03-2a51639fc7d5
+```
+
+#### Sample response
+
+```
+{
+  "data": {
+    "contacts": null,
+    "created_time": "2024-12-12T12:55:50-05:00",
+    "directory": "acme/",
+    "eab": {
+      "created_time": "2024-12-12T12:55:29-05:00",
+      "directory": "acme/",
+      "eab_id": "24c0673a-df53-0671-a628-e7b9c995485c",
+      "key_type": "hs"
+    },
+    "key_id": "2ea9859a-eba8-ff24-cd03-2a51639fc7d5",
+    "orders": [
+      {
+        "cert_expiry": "2024-12-13T17:55:28Z",
+        "cert_serial_number": "4a:6f:d0:f7:13:55:f7:c9:19:82:fc:34:69:67:77:2e:58:27:02:8b",
+        "identifiers": [
+          "testing.dadgarcorp.com"
+        ],
+        "order_expiry": "2024-12-13T12:56:04-05:00",
+        "order_id": "90699994-8863-571c-26b0-46755e0db351",
+        "status": "valid"
+      }
+    ],
+    "revoked_time": "",
+    "status": "valid"
+  },
+}
+```
+
+### Update ACME Account Info
+
+This endpoint allows updating an ACME account's status to revoke or re-enable it by
+a Vault operator. The returned response contains the ACME account information without,
+any order or certificate details.
+
+| Method | Path                                   |
+|:-------|:---------------------------------------|
+| `POST` | `/pki/acme/mgmt/account/keyid/:key_id` |
+
+#### Parameters
+
+- `key_id` `(string: <required>)` - The id of the ACME account to read.
+  This is part of the request URL.
+
+- `status` `(string: <required>)` - The status to update the account to, allowed
+  values are `revoked` and `valid`.
+
+ ~> Note: Setting the status to `revoked` will forbid any further operations
```

Comment:
FWIW, the docs team appears to be favouring `<Note>` tags now, see for example https://github.com/hashicorp/vault/pull/29136/files

Reply:
TIL, fixed in https://github.com/hashicorp/vault/pull/29173/commits/6427ec24292557f70ab67475ec2d369a34bc086d